### DISCUSSION TITLE
Spec semantic versioning

### DISCUSSION THREAD

**Discussion by sevetseh28** [devcontainers/spec#457] [General]
Hi, I wanted to initiate a discussion, as I realised the spec doesn't have any semantic versioning. Instead, the changes are directly pushed to the repo.

I'm wondering what if there's any backwards incompatible change or a new feature added? How do we know that the devcontainer CLI we are using supports the new spec version?

thank you!

**Comment by samruddhikhandale** [comment]
Hi 👋 

I believe we have https://github.com/devcontainers/spec/issues/248 open to gather more feedback on spec versioning. 
Do you mind sharing your thoughts on the issue? Thanks! 

As dev container spec has evolved and grown since it became open specification, I think we should focus on this topic and add semantic versioning. // cc @bamurtaugh

**Reply by sevetseh28** [reply]
Hey! Thanks for your answer. I'll take a look and participate on the discussion !
